.github/workflows/integration-tests.yml

```diff
@@ -19,14 +19,27 @@ jobs:
         fetch-depth: 0
     - name: Setup PR environment
       if: inputs.trigger == 'pull-request'
-      run: | # If the image with PR branch's version as tag doesn't exist, use the one in main that should already exist
+      # If the image with PR branch's version as tag doesn't exist, use the one in main that should already exist.
+      # If main's doesn't exist, that means there was an official release, so the short version is chosen
+      run: |
         git fetch -q origin main:main
-        docker manifest inspect ${IMAGE_REGISTRY}/kuberpult-cd-service:$(make version) > /dev/null && \
-          echo "IMAGE_TAG_CD=$(make version)" >> $GITHUB_ENV || \
+
+        if docker manifest inspect ${IMAGE_REGISTRY}/kuberpult-cd-service:$(make version) > /dev/null; then
+          echo "IMAGE_TAG_CD=$(make version)" >> $GITHUB_ENV
+        elif docker manifest inspect ${IMAGE_REGISTRY}/kuberpult-cd-service:$(git describe --always --long --tags main) > /dev/null; then
           echo "IMAGE_TAG_CD=$(git describe --always --long --tags main)" >> $GITHUB_ENV
-        docker manifest inspect ${IMAGE_REGISTRY}/kuberpult-frontend-service:$(make version) > /dev/null && \
-          echo "IMAGE_TAG_FRONTEND=$(make version)" >> $GITHUB_ENV || \
+        elif docker manifest inspect ${IMAGE_REGISTRY}/kuberpult-cd-service:$(git describe --always --tags main) > /dev/null; then
+          echo "IMAGE_TAG_CD=$(git describe --always --tags main)" >> $GITHUB_ENV
+        fi
+
+        if docker manifest inspect ${IMAGE_REGISTRY}/kuberpult-frontend-service:$(make version) > /dev/null; then
+          echo "IMAGE_TAG_FRONTEND=$(make version)" >> $GITHUB_ENV
+        elif docker manifest inspect ${IMAGE_REGISTRY}/kuberpult-frontend-service:$(git describe --always --long --tags main); then
           echo "IMAGE_TAG_FRONTEND=$(git describe --always --long --tags main)" >> $GITHUB_ENV
+        elif docker manifest inspect ${IMAGE_REGISTRY}/kuberpult-frontend-service:$(git describe --always --tags main); then
+          echo "IMAGE_TAG_FRONTEND=$(git describe --always --tags main)" >> $GITHUB_ENV
+        fi
```

Comment:
what if none of the if-conditions match?
Please add an "else". I imagine in the "else" we want to exit with an error.